dbus/com.redhat.Yggdrasil1.xml

```diff
@@ -62,6 +63,7 @@
             <arg type="s" name="worker" />
             <arg type="u" name="name" />
             <arg type="s" name="message_id" />
+            <arg type="s" name="response_to" />
```

Comment:
We can avoid breaking *this* API by putting the `response_to` field below `message`.